functions/go/apply-setters/README.md

```diff
@@ -0,0 +1,94 @@
+# apply-setters
```

Comment:
I assume this doc has been reviewed in the kpt repo.

Reply:
I suggest reviewing it again. Also need to think about standardizing the function usage across all 1st party functions. Can you work with @phanimarupaka to define a template readme (Similar to how we have examples/template).

Reply:
Sure

Reply:
It is not.